<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SwitchSurfaces</class>
 <widget class="QDialog" name="SwitchSurfaces">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Switch surfaces</string>
  </property>
  <property name="windowIcon">
   <iconset resource="icons.qrc">
    <normaloff>:/mk/shared/editor.ico</normaloff>:/mk/shared/editor.ico</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Surface 1</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Surface 2</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="inputSurface_1">
       <property name="toolTip">
        <string>Replace this surface…</string>
       </property>
       <item>
        <property name="text">
         <string>1 - Grass</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/brush_surface_grass1.png</normaloff>:/icons/brush_surface_grass1.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 - Secondary grass</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/brush_surface_grass2.png</normaloff>:/icons/brush_surface_grass2.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 - Road</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/brush_surface_road.png</normaloff>:/icons/brush_surface_road.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 - Stone</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/brush_surface_stone.png</normaloff>:/icons/brush_surface_stone.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 - Ground</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/brush_surface_custom.png</normaloff>:/icons/brush_surface_custom.png</iconset>
        </property>
       </item>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QComboBox" name="inputSurface_2">
       <property name="toolTip">
        <string>…with this and vice versa</string>
       </property>
       <property name="currentIndex">
        <number>1</number>
       </property>
       <item>
        <property name="text">
         <string>1 - Grass</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/brush_surface_grass1.png</normaloff>:/icons/brush_surface_grass1.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>2 - Secondary grass</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/brush_surface_grass2.png</normaloff>:/icons/brush_surface_grass2.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>3 - Road</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/brush_surface_road.png</normaloff>:/icons/brush_surface_road.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>4 - Stone</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/brush_surface_stone.png</normaloff>:/icons/brush_surface_stone.png</iconset>
        </property>
       </item>
       <item>
        <property name="text">
         <string>5 - Ground</string>
        </property>
        <property name="icon">
         <iconset resource="icons.qrc">
          <normaloff>:/icons/brush_surface_custom.png</normaloff>:/icons/brush_surface_custom.png</iconset>
        </property>
       </item>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>inputSurface_1</tabstop>
  <tabstop>inputSurface_2</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SwitchSurfaces</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>226</x>
     <y>285</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SwitchSurfaces</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>294</x>
     <y>291</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
